tests/pages.test.js

```diff
@@ -1,43 +1,68 @@
+import { describe, expect, test } from "@jest/globals";
+import { marked } from "marked";
+import { loadMovies } from "../src/movies.js";
 import { loadMovie } from "../src/movies.js";
```

Comment:
Don't need two lines with imports from the same file! 😁

Reply:
Fixed :D